### DISCUSSION TITLE
Support line-based comment.

### DISCUSSION THREAD

**Discussion by cathaysia** [mitsuhiko/minijinja#562] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/057853b9-aee7-4f72-9132-163d29f23332)

Jinja2 support line comment, does minijinja will support it?

**Comment by mitsuhiko** [comment, marked as answer]
Yes, if you enable the `custom_syntax` feature you can configure line comments:

```rust
use minijinja::{context, syntax::SyntaxConfig, Environment};

let mut env = Environment::new();
env.set_syntax(
    SyntaxConfig::builder()
        .line_statement_prefix("#")
        .line_comment_prefix("##")
        .build()
        .unwrap(),
);
```

**Reply by cathaysia** [reply]
Why isn't this functionality built in?

**Reply by cathaysia** [reply]
oh, sorry, I missed this:
![image](https://github.com/user-attachments/assets/04c69119-6ac2-4cef-850c-8372043bd5bc)

**Reply by mitsuhiko** [reply]
It is built in. You just need to enable the feature flag. Like you noticed in neither Jinja2 nor minijinja is it enabled by default.
